packages/dvilela/skills/memeooorr_abci/behaviour_classes/chain.py

```diff
@@ -326,11 +326,6 @@ def get_tx_hash(self) -> Generator[None, None, Optional[str]]:
             value=value,
         )
 
-        # Optimistic design: we now store the hearted token address
-        # Ideally, this should be done after a succesful heart transaction
-        if action == "heart":
-            self.store_heart(token_action["token_nonce"])
-
```

Comment:
Does not have to do with the bump, it was just in my stash and I wanted to merge this minor change.